launcher/pom.xml

```diff
@@ -73,6 +73,11 @@
         </dependency>
         <dependency>
             <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-cm-salt</artifactId>
```

Comment:
launcher should be quite bare.  i think the `all` project is where things like this are brought in.